src/lips.js

```diff
@@ -6920,11 +6920,11 @@ function interaction(env, name) {
 // -------------------------------------------------------------------------
 var internal_env = new Environment({
     stdout: new BufferedOutputPort(function(...args) {
-        console.log(...args);
+        console.log(...args); // @jcubic can you enter what this console log returns here
```

Comment:
This function is part of the STDOUT output of the Scheme code. If you run `(display "hello")` this is the default function that is called.

Reply:
Should be deleted now. I made a new commit.